bin/gen/config.yaml

```diff
@@ -11,6 +11,10 @@ examples:
   text:    "BitTorrent metainfo related functionality is under the `torrent` subcommand:"
   code:    "imdl torrent --help"
 
+- command: imdl torrent announce
```

Comment:
I suspect that the primary initial use for this command will be to get the printed peer list, so that should be mentioned. Like, "Announce a torrent to its trackers and print all peers returned:"